src/plone/api/content.py

```diff
@@ -100,7 +100,7 @@ def create(
             '{message}'.format(
                 obj_type=type,
                 allowed_types='\n'.join(sorted(types)),
-                message=e.message,
+                message=e.args[0],
```

Comment:
AFAIK this should be fixed with `str(e)`; see: https://stackoverflow.com/a/7075416